airflow/www/views.py

```diff
@@ -1051,10 +1051,12 @@ def cluster_activity(self):
         """Cluster Activity view."""
         state_color_mapping = State.state_color.copy()
         state_color_mapping["no_status"] = state_color_mapping.pop(None)
+        standalone_dag_processor = str(conf.getboolean("scheduler", "standalone_dag_processor")).lower()
```

Comment:
@bbovenzi I had to convert to string and lowercase this value, but I'm not sure if this covers all possible values for `standalone_dag_processor` like if it equals 0 or 1. Not sure how these types of situations with booleans are typically handled in the codebase.

Reply:
Given other sppts where we pass this. I think we can just pass the regular value and js will interpret it as a string of "True" or "False"